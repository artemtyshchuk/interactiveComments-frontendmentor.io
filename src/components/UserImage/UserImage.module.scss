.userImage {
  width: 40px;
  height: 40px;
  border-radius: 40px;
}

.userImageReply {
  width: 32px;
  height: 32px;
  border-radius: 32px;
}
