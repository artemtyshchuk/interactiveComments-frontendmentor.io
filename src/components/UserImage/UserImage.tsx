import styles from "./UserImage.module.scss";

interface UserImageProps {
  image: string;
  alt: string;
  inputImage?: boolean;
}
export const UserImage = ({ image, alt, inputImage }: UserImageProps) => {
  return (
    <img
      src={image}
      alt={alt}
      className={`${
        inputImage ? styles.userImage : styles.userImageReply
      }`}
    />
  );
};
