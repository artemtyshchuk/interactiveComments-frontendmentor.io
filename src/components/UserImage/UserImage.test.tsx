import { render, screen } from "@testing-library/react";
import { UserImage } from "./UserImage";

describe("UserImage", () => {
  it("should render correctly", () => {
    render(<UserImage image="image" alt="alt" />);

    expect(screen.getByAltText("alt")).toHaveAttribute("src", "image");
    expect(screen.getByAltText("alt")).toMatchSnapshot();
  });
});
