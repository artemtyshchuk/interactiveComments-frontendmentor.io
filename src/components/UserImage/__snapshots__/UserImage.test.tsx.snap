// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserImage should render correctly 1`] = `
<img
  alt="alt"
  class="userImageReply"
  src="image"
/>
`;
